python-libraries/data-platform-catalogue/data_platform_catalogue/client/datahub/datahub_client.py

```diff
@@ -151,41 +151,51 @@ def upsert_data_product(self, metadata: DataProductMetadata):
 
         data_product_urn = "urn:li:dataProduct:" + "".join(name.split())
 
-        if metadata.domain:
-            domain = metadata_dict.pop("domain")
-            domain_urn = self.graph.get_domain_urn_by_name(domain_name=domain)
+        domain = metadata_dict.pop("domain")
+        domain_urn = self.graph.get_domain_urn_by_name(domain_name=domain)
```

Comment:
Do we want to sentence case or lowercase `domain`? Is `get_domain_urn_by_name` case sensitive?

Reply:
Domains are currently hard coded in the front end, users dont have the ability to enter them.